modules/installation-supported-aws-machine-types.adoc

```diff
@@ -23,10 +24,14 @@ ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
 
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
 
 [id="installation-supported-aws-machine-types_{context}"]
 = Supported AWS machine types
```

Comment:
We use `Tested` [1] instead of `Supported` [2] in the official doc.



[1] https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-localzone.html#installation-aws-tested-machine-types_installing-aws-localzone
[2] https://57057--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-localzone.html#installation-supported-aws-machine-types_installing-aws-localzone

Reply:
Hi @mtulio . We need to define "tested" here and not "supported"?

Reply:
I interpreted the following conversation to state "supported"? 

https://github.com/openshift/openshift-docs/pull/54535/files#r1068870303

Reply:
@dfitzmau `tested` is being used by all cloud providers:

* AWS https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-customizations.html#installation-aws-tested-machine-types_installing-aws-customizations

* Azure https://docs.openshift.com/container-platform/4.12/installing/installing_azure/installing-azure-customizations.html#installation-azure-tested-machine-types_installing-azure-customizations

* GCP https://docs.openshift.com/container-platform/4.12/installing/installing_gcp/installing-gcp-network-customizations.html#installation-gcp-tested-machine-types_installing-gcp-network-customizations

Reply:
Thanks, @yunjiang29 . I moved the content back to a "Tested" module. I had to revert the introductory content, because the file is used by other installation routes (proclaimed by ifdef and endif statements).

Please let me know your thoughts.